webapi/scripts/openrdf-console/bin/win-console.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
This file is in a folder which is not managed by us. From time to time, we exchange the `openrdf-console` folder with a current version of the distribution, which will remove this file.

Reply:
Ok, I see. What do you recommend? Moving the win-file to `/scripts` or replacing it every time the `openrdf-console` is exchanged?

Reply:
I have downloaded the newest version and now they have now a `console.bat`, so it should work under windows. So shouldn't be a problem when we exchange this folder.

Reply:
That's nice. Should we use the `console.bat` right now, so no further change is needed? But I can't see this file, wether on this branch nor on `develop`. If not, I can also merge now and change later on.

Reply:
I didn't update the directory. I don't have time now to test all scripts.